Dshop/apps/products_catalogue/tests.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+from .models import Category, SubCategory
+
+
+class TestCategoriesModels(TestCase):
```

Comment:
Please follow roadmap: https://github.com/akademiait/Dshop/blob/dev/roadmap.md
Use pytest for testing. 

```
pytest
```
Examples: https://github.com/jacoor/django_testing_examples/blob/master/accounts/tests/tests_accounts_forms.py

I know, django has great test engine. Pytest is more universal and people will be able to use the same framework with Django or Flask or whatever.